actions/ostree_commit_action.go

```diff
@@ -1,3 +1,28 @@
+/*
+OstreeCommit Action
+
+Create OSTree commit from rootfs prepared with other actions.
```

Comment:
The prepared with other actions is redundant fwiw

Reply:
done